packages/back-end/src/integrations/Mssql.ts

```diff
@@ -21,7 +21,7 @@ export default class Mssql extends SqlIntegration {
     return ["password"];
   }
   async runQuery(sqlStr: string) {
-    const conn = await mssql.connect({
+    const conn = await get(this.params.database, {
```

Comment:
I wonder if we should index by the display name instead (or a normalized version of it). It could be a valid use case to have multiple connections to the same database (different users/permissions).

If the display name isn't available, maybe a composite key of `database-name_user` would do.

Reply:
@bttf Agreed - display name isn't available, but the datasourceId is, and we shouldn't run into duplicates that way.

Reply:
Yeah one connection pool per datasource seems like the right way to go and referencing by ID seems best. New version looks good.